api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -506,6 +507,22 @@ def serialize_offer(cls, offer: offers_models.Offer, last_30_days_bookings: int)
             gtl_code_3 = gtl_id[:6] + "00" * 1
             gtl_code_4 = gtl_id
 
+        address = None
+        city = None
+        departmentCode = None
+        postalCode = None
```

Comment:
Détail : Les conventions dans le code du pass Culture font qu'on n'utilise pas de noms de variable en camelCase. C'est le cas aussi dans d'autres fonctions plus bas.
```suggestion
        department_code = None
        postal_code = None
```

Reply:
Effectivement, j'ai fait du copier coller des champs tels qu'ils sont dans les serializers.
Il faudrait que l'on nettoie nos models et serializers pour avoir tout en snakecase !